core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/controlplane/services/transferprocess/TransferProcessProtocolServiceImpl.java

```diff
@@ -233,11 +233,21 @@ private ServiceResult<TransferProcess> suspendedAction(TransferSuspensionMessage
     @NotNull
     private ServiceResult<TransferProcess> terminatedAction(TransferTerminationMessage message, TransferProcess transferProcess) {
         if (transferProcess.canBeTerminated()) {
+			if (transferProcess.getType() == PROVIDER) {
+				var termination = dataFlowManager.terminate(transferProcess);
+				if (termination.failed()) {
+					return ServiceResult.conflict("Cannot terminate transfer process %s: %s".formatted(transferProcess.getId(), termination.getFailureDetail()));
+				}
+			}
```

Comment:
spacing is still misplaced. checkstyle will fail, but I guess this is something that could be fixed further on

Reply:
Strange, I just do ctrl+x and ctrl+v and spaces was corrected, but visually nothing changed in the IDE.